### PR TITLE
remove unused devDependencies

### PR DESCRIPTION
**Pull request by LiKang6688**

### CODE REVIEW

**Review by haxxmaxx — APPROVED** [review]

**Comment by LiKang6688** [comment]
@haxxmaxx  Can I merge now? or Do we need Tobias and Henrik's approval as well?

**Comment by haxxmaxx** [comment]
No need for more approvals, but the test are not running no Circle CI today for some reason, we want those to pass before merging

**Review by Caele — APPROVED** [review]

**Pull request merged**
